integrations/cloudbuild/smoke-test.yaml

```diff
@@ -1,5 +1,5 @@
 steps:
-    - name: "connectedhomeip/chip-build-vscode:0.5.58"
```

Comment:
0.5.64 contains a NRF update for NCS revision: https://docs.google.com/spreadsheets/d/14wVVdeQpIGKQZnbvWD52EZeJGPn-05Lqk7oKNneHsA0/edit#gid=0

I believe we need to change config/nrfconnect/.nrfconnect-recommended-revision otherwise we will get compile errors in cloudbuild (those scripts only check but do not update the revision)

Reply:
#17236 will update NRF NCS version. Need to sequence the changes.